dotnet/src/Extensions/Planning.StepwisePlanner/StepwisePlanner.cs

```diff
@@ -58,9 +58,6 @@ public class StepwisePlanner : IStepwisePlanner
         // Load or use default PromptConfig
         this._promptConfig = this.Config.PromptUserConfig ?? LoadPromptConfigFromResource();
 
-        // Set MaxTokens for the prompt config
```

Comment:
I still think we need to be doing this somehow. And also maybe an opportunity to help define something missing in the kernel. The ability to defin 1) how big a prompt is allowed to get before submission and 2) the amount of tokens you want generated. As this has been lacking, the implementation now just uses the same value for each with the assumption that the user configures it to be at most half the allowable token limit of the model being used. This could be done separately, too. Just getting the thoughts out.

Reply:
I'm going to add back in the code that will provide the equivalent functionality so we don't have a regression. 

I agree we do need to have well defined patterns to allow the appropriate default request settings to be provided. I am working on a plan for this and will have a PR + ADR in the next week or so.